tests/main/snap-wait/task.yaml

```diff
@@ -10,6 +10,9 @@ execute: |
     echo "Ensure snap wait for seeding works"
     snap wait system seed.loaded
 
+    echo "Ensure snapd.seeded.service is active"
+    systemctl status snapd.seeded.service | MATCH "SUCCESS"
```

Comment:
You could just do `if ! systemctl is-active snapd.seeded.service; then ...`. It returns 0 on success...